cinema/serializers.py

```diff
@@ -1,6 +1,5 @@
 from rest_framework import serializers
-
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
 class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for `MovieSerializer`. This will simplify the code by automatically generating fields and methods based on the `Movie` model.